app/core/Router.coffee

```diff
@@ -72,15 +72,15 @@ module.exports = class CocoRouter extends Backbone.Router
     'contribute/diplomat': go('contribute/DiplomatView')
     'contribute/scribe': go('contribute/ScribeView')
 
-    'courses': go('courses/CoursesView')
-    'Courses': go('courses/CoursesView')
-    'courses/students': redirect('/courses')
+    'courses': redirect('/students') # Redirected 9/3/16
+    'Courses': redirect('/students') # Redirected 9/3/16
```

Comment:
Testing on localhost, if I go to `/courses?_cc=test`, the query parameter is lost in the redirect. Looks like `redirect` needs to be set up to keep the query string.